index.html

```diff
@@ -57,6 +57,132 @@ <h1 class="intro">I’m Selase 👋 Glad to see you!</h1>
+                    <div class="role color-gray">Back End Dev</div>
+                    <div class="year color-gray">2015</div>
+                </div>
+                <p class="details">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <div class="tags">
+                    <p class="tag">html</p>
+                    <p class="tag">css</p>
+                    <p class="tag">javascript</p>
+                </div>
+                <button class="btn">See Project</button>
+            </article>
+
+            <article class="card">
+                <img src="./images/tonic-2.png" alt="" aria-hidden="true">
+                <h3 class="title color-caption">Tonic</h3>
+                <div class="profile">
+                    <div class="client">Canopy</div>
+                    <div class="role color-gray">Back End Dev</div>
+                    <div class="year color-gray">2015</div>
+                </div>
+                <p class="details">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <div class="tags">
+                    <p class="tag">html</p>
+                    <p class="tag">css</p>
+                    <p class="tag">javascript</p>
+                </div>
+                <button class="btn">See Project</button>
+            </article>
+
+            <article class="card">
+                <img src="./images/stories-2.png" alt="" aria-hidden="true">
+                <h3 class="title color-caption">Multi-Post
+                    Stories</h3>
+                <div class="profile">
+                    <div class="client">Canopy</div>
+                    <div class="role color-gray">Back End Dev</div>
+                    <div class="year color-gray">2015</div>
+                </div>
+                <p class="details">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <div class="tags">
+                    <p class="tag">html</p>
+                    <p class="tag">css</p>
+                    <p class="tag">javascript</p>
+                </div>
+                <button class="btn">See Project</button>
+            </article>
+        </section>
+
+        <section class="about-self wrapper">
+            <h2 class="sr-only">About Section</h2>
+            <h3 class="intro about-intro color-caption">About
+                Myself</h3>
+            <p class="description">
+                I’m a software developer! I can help you build a product , feature or website. Look through some of
+                my work and experience! If you like what you see and have a project you need coded, don’t hestiate
+                to contact me.
+            </p>
+            <p class="social-media">Let's connect</p>
```

Comment:
Kindly use `h1-6` tags instead of `p` tag, it's a kind of title so it's more semantic to use headline tags. this one `<p class="social-media">Let's connect</p>`